packages/salesforcedx-vscode-core/src/commands/forceConfigSet.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  Command,
+  SfdxCommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+
+import {
+  EmptyParametersGatherer,
+  SfdxCommandlet,
+  SfdxCommandletExecutor,
+  SfdxWorkspaceChecker
+} from './commands';
+
+export class ForceConfigSetExecutor extends SfdxCommandletExecutor<{}> {
+  private arg: string;
+
+  public constructor(arg: string) {
+    super();
+    this.arg = arg;
+  }
+
+  public build(data: {}): Command {
+    return new SfdxCommandBuilder()
+      .withDescription('SFDX: Set a Default Org')
+      .withArg('force:config:set')
+      .withArg('defaultusername=' + this.arg)
```

Comment:
Let's change this from string concatenation to template literals (`defaultusername=${this.arg}`)

Reply:
Yeah, sounds good.